lib/trento_web/router.ex

```diff
@@ -64,6 +64,12 @@ defmodule TrentoWeb.Router do
          ClusterController,
          :request_checks_execution
 
+    get "/clusters/:cluster_id/connection-settings", ClusterController, :get_connection_settings
+
+    post "/clusters/:cluster_id/connection-settings",
```

Comment:
Being purist, I see this more like a `put` rather than a `post`, as we do not create new entries most of the cases, we just update them

Reply:
I see the point, changed.

So, are we gonna be purists? Fine with me :trollface: